src/rest_api/formatters/object.clj

```diff
@@ -160,18 +161,17 @@
   ;; computing the display name.
   (let [db (d/entity-db int)]
     (if-let [il (seq (d/q q-interactor db (:db/id int)))]
```

Comment:
@a8wright  I've [made][0f02db0dedf72b26f21288954907c0ab63eeb639] the changes you requested,  is this good for you now?